src/app/App.jsx

```diff
@@ -1,18 +1,49 @@
 import { Routes, Route, Navigate } from "react-router-dom";
+import { useState, useEffect } from "react";
 
+import authUser from "../utils/authUser";
 import Login from "../components/pages/Login";
 import Header from "../components/Header";
+import Dashboard from "../components/pages/Dashboard";
 import Sidebar from "../components/Sidebar";
 
 function App() {
+  const [user, setUser] = useState("");
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+
+  useEffect(() => {
+    const checkLoginStatus = async () => {
```

Comment:
const checkLoginStatus = async () => {
```suggestion
    async function checkLoginStatus() {
```
 
 컨벤션 통일 부탁드립니다!

Reply:
앗 유진님 나이스 캐치!! 
바로 수정하겠습니다! 감사합니다!